src/components/keyboard-handler/index.js

```diff
@@ -3,6 +3,7 @@ import { useKeyboardContext } from '@store/keyboard/context';
 import { ALT_KEY } from '@constants/AppConstants';
 
 function isOptionKey(e) {
+  if (e.key === ALT_KEY) e.preventDefault();
   return e.key === ALT_KEY;
```

Comment:
so if we are already checking this condition, why don't we just return true inside the code block only